erpnext/controllers/taxes_and_totals.py

```diff
@@ -438,37 +440,46 @@ def calculate_taxes(self):
 					tax.grand_total_for_current_item = flt(item.net_amount + current_tax_amount)
 				else:
 					tax.grand_total_for_current_item = flt(
-						self.doc.get("taxes")[i - 1].grand_total_for_current_item + current_tax_amount
+						doc.taxes[i - 1].grand_total_for_current_item + current_tax_amount
 					)
 
-				# set precision in the last item iteration
-				if n == len(self._items) - 1:
-					self.round_off_totals(tax)
-					self._set_in_company_currency(
-						tax, ["tax_amount", "tax_amount_after_discount_amount", "net_amount"]
-					)
+				logger.debug(
```

Comment:
Is the logging intentional? or forgot to remove?

Reply:
I have not added it. This was added in https://github.com/frappe/erpnext/pull/44229/.

I can remove all the calls though.

Reply:
> I have not added it. This was added in #44229.
> 
> I can remove all the calls though.

I'll remove this one for now. Rest can be dealt with in a separate PR.

Reply:
Sorry, I removed them all 😅

Reply:
It's a separate commit though.

Reply:
Fine